core/data_migration_scripts/EE_DMS_Core_4_6_0.dms.php

```diff
@@ -374,126 +374,126 @@ public function schema_changes_before_migration()
 					  KEY REG_code (REG_code)";
         $this->_table_should_exist_previously($table_name, $sql, 'ENGINE=InnoDB ');
         $table_name = 'esp_checkin';
-        $sql = "CHK_ID INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
-					REG_ID INT(10) UNSIGNED NOT NULL,
-					DTT_ID INT(10) UNSIGNED NOT NULL,
-					CHK_in TINYINT(1) UNSIGNED NOT NULL DEFAULT 1,
-					CHK_timestamp DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
+        $sql = "CHK_ID int(10) unsigned NOT NULL AUTO_INCREMENT,
+					REG_ID int(10) unsigned NOT NULL,
+					DTT_ID int(10) unsigned NOT NULL,
+					CHK_in tinyint(1) unsigned NOT NULL DEFAULT 1,
+					CHK_timestamp datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
 					PRIMARY KEY  (CHK_ID)";
         $this->_table_should_exist_previously($table_name, $sql, 'ENGINE=InnoDB');
         $table_name = 'esp_state';
-        $sql = "STA_ID smallint(5) UNSIGNED NOT NULL AUTO_INCREMENT,
-					  CNT_ISO VARCHAR(2) COLLATE utf8_bin NOT NULL,
-					  STA_abbrev VARCHAR(24) COLLATE utf8_bin NOT NULL,
-					  STA_name VARCHAR(100) COLLATE utf8_bin NOT NULL,
-					  STA_active TINYINT(1) DEFAULT '1',
+        $sql = "STA_ID smallint(5) unsigned NOT NULL AUTO_INCREMENT,
+					  CNT_ISO varchar(2) COLLATE utf8_bin NOT NULL,
+					  STA_abbrev varchar(24) COLLATE utf8_bin NOT NULL,
+					  STA_name varchar(100) COLLATE utf8_bin NOT NULL,
+					  STA_active tinyint(1) DEFAULT '1',
 					  PRIMARY KEY  (STA_ID)";
         $this->_table_should_exist_previously($table_name, $sql, 'ENGINE=InnoDB');
         $table_name = 'esp_status';
-        $sql = "STS_ID VARCHAR(3) COLLATE utf8_bin NOT NULL,
-					  STS_code VARCHAR(45) COLLATE utf8_bin NOT NULL,
+        $sql = "STS_ID varchar(3) COLLATE utf8_bin NOT NULL,
+					  STS_code varchar(45) COLLATE utf8_bin NOT NULL,
 					  STS_type set('event','registration','transaction','payment','email') COLLATE utf8_bin NOT NULL,
-					  STS_can_edit TINYINT(1) NOT NULL DEFAULT 0,
-					  STS_desc TINYTEXT COLLATE utf8_bin,
-					  STS_open TINYINT(1) NOT NULL DEFAULT 1,
+					  STS_can_edit tinyint(1) NOT NULL DEFAULT 0,
+					  STS_desc TINYtext COLLATE utf8_bin,
```

Comment:
Should this be `tinytext`?

Reply:
Probably would be worthwhile doing a case-sensitive search for `TINYtext` via your IDE to catch all occurrences of this as I'll probably miss some in my review).  So I won't comment anymore on this particular thing.

Reply:
ya, no more occurrences of it, thanks